docs/sphinx/examples/cpp/providers/photonics_tbi_get_state.cpp

```diff
@@ -0,0 +1,87 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+
+// Compile and run with:
+// ```
+// nvq++ --target photonics-cpu photonics_tbi_get_state.cpp -o
```

Comment:
nit: shouldn't this have a `\` at the end of the line?

Reply:
`&&` was missing